src/bin/edit/main.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-#![feature(let_chains, linked_list_cursors, os_string_truncate, string_from_utf8_lossy_owned)]
+#![feature(let_chains, linked_list_cursors, os_string_truncate, string_from_utf8_lossy_owned, variant_count)]
```

Comment:
We also received very rightful criticism that we shouldn't depend on too many nightly features. As such, I think we should back out this particular change.